islandora_ocr.install

```diff
@@ -36,13 +36,19 @@ function islandora_ocr_enable() {
  */
 function islandora_ocr_cleanup_derivatives() {
   if (module_exists('islandora_book')) {
-    $book = variable_get('islandora_book_ingest_derivatives');
+    $book = variable_get('islandora_book_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $book['ocr'] = FALSE;
     variable_set('islandora_book_ingest_derivatives', $book);
   }
 
   if (module_exists('islandora_newspaper')) {
-    $newspaper = variable_get('islandora_newspaper_ingest_derivatives');
+    $newspaper = variable_get('islandora_newspaper_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $newspaper['ocr'] = FALSE;
```

Comment:
This just needs the second 'false' to be removed, right? for newspaper and ocr? @whikloj that was your comment right?